mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,17 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrderById = (req, res) => {
+  Order.deleteOne({ _id: req.params.orderId })
+    .exec()
+    .then(result => {
+      if (result.n === 0) {
+        return res.status(404).json({ message: "Order not found" });
+      }
+      res.status(200).json({ message: "Order successfully removed" });
```

Comment:
The response message "Order successfully removed" should be consistent throughout the application. It's better to define it as a constant in a separate file, e.g., `constants.js`, and import it where needed. This will ensure consistency and make it easier to change the message in the future.